source/api_documentation/transactions_api/advertiser_user.rst

```diff
@@ -26,6 +26,8 @@ The API follows REST conventions. Perform an HTTPS GET to the URL with the forma
 
 
 .. include:: _intro.rst
+.. include:: _intro_constants.rst
+
```

Comment:
Looks like an extra new line here (not sure if this affects the resulting formatting)